Objects/setobject.c

```diff
@@ -1770,7 +1770,11 @@ set_issubset(PySetObject *so, PyObject *other)
     Py_RETURN_TRUE;
 }
 
-PyDoc_STRVAR(issubset_doc, "Report whether another set contains this set.");
+PyDoc_STRVAR(issubset_doc,
+"issubset($self, /, other)\n\
```

Comment:
Is `other` a positional-or-keyword parameter?

Reply:
> Is `other` a positional-or-keyword parameter?

Thanks for reminding, `other` is a positional-only parameter. I made a mistake here.